articles/techniques/testing-and-debugging.md

```diff
@@ -256,7 +256,7 @@ Notice how the IDs of the qubits are shown at the top in their significant order
 ***
 
 
-@"microsoft.quantum.extensions.diagnostics.dumpmachine" is part of the  <xref:microsoft.quantum.extensions.diagnostics> namespace, so in order to use it you must add an `open` statement:
+@"microsoft.quantum.extensions.diagnostics.dumpmachine" is part of the  <xref:microsoft.quantum.diagnostics> namespace, so in order to use it you must add an `open` statement:
```

Comment:
```suggestion
<xref:microsoft.quantum.diagnostics.dumpmachine> is part of the  <xref:microsoft.quantum.diagnostics> namespace, so in order to use it you must add an `open` statement:
```

Reply:
It looks like the reference is still using `@""` and `extensions` rather than `<xref:>` syntax and the new namespace name?